### DISCUSSION TITLE
Kubernetes deployment

### DISCUSSION THREAD

**Discussion by belov38** [evershopcommerce/evershop#562] [Feature Requests & Ideas]
Hey Team,

Any plans to create k8s deployment (like helm chart or so) for your awsome project?

**Comment by xyphanajay** [comment]
Let me try to come up with one with @akashlakhera

**Comment by VictorTrapenok** [comment]
Hi.
I created a chart https://artifacthub.io/packages/helm/unifie/evershop
Please let me know if needed any fixes there.
